test/powershell/Modules/Microsoft.PowerShell.Management/Add-Content.Tests.ps1

```diff
@@ -35,6 +35,10 @@ Describe "Add-Content cmdlet tests" -Tags "CI" {
       { Add-Content -Path $null -Value "ShouldNotWorkBecausePathIsNull" -ErrorAction Stop } | Should -Throw -ErrorId "ParameterArgumentValidationErrorNullNotAllowed,Microsoft.PowerShell.Commands.AddContentCommand"
     }
 
+    It "should throw 'GetContainerContentException' when -Path is a container" {
+      { Add-Content -Path . -Value "GetContainerContentException" -ErrorAction Stop } | Should -Throw -ErrorId "GetContainerContentException,Microsoft.PowerShell.Commands.AddContentCommand"
```

Comment:
For "Add" in cmdlet name the "GetContainerContentException" does not look very good.